doc: wifi: Move Regulatory Certification Testing doc to NCS (#21024)
nrfconnect/sdk-nrf: doc/nrf/protocols/wifi/regulatory_certification/wifi_radio_test_sample/wifi_radio_sample_for_transmit_tests.rst

.. _ug_wifi_radio_sample_for_transmit_tests:

Using the Wi-Fi Radio sample for transmit tests
###############################################

.. contents::
   :local:
   :depth: 2

Set the transmit parameters and modes with the correct order of commands.
The following examples show the input and sequence of commands for the different Wi-Fi® modes.

Order of commands
*****************

The order of the Wi-Fi Radio sub-commands is very important.

Use the following order of sub-commands:

1. ``init`` – Disables any ongoing TX or RX testing and sets all configured parameters to default.
#. ``tx_pkt_tput_mode`` – Sets the frame format of the transmitted packet.

   other commands...
#. ``ru_tone`` – Configures the Resource Unit (RU) size.
#. ``ru_index`` – Configures the location of RU in 20 MHz spectrum.
#. ``tx_pkt_len`` – Packet data length to be used for the TX stream.

   other commands...
#. ``tx`` – Starts transmit.

   .. note::

>       The command to start transmit, ``tx 1``, must only be given after all the parameters are configured.

Reviewer: ```suggestion
      Use the command to start transmit (``tx 1``) only after all the parameters are configured.
```

The last sub-command:

.. code-block:: shell

   uart:~$ wifi_radio_test tx 1

Adjusting TX power
******************

For any Wi-Fi mode and modulation, the regulatory domain control is enabled as a default setting where the TX power defaults to :math:`TX_\text{max}`.
This is the lower of the nRF70 Series Product Specification value and the maximum value set by the regulatory domain for the specified channel.

To set the transmission power to a specific value, regardless of product specification or regulatory domain limits, you must first disable the regulatory domain settings.
Then, use the ``tx_power`` command to set the required transmit power.
``<X>`` is the transmit power expressed in dBm in the range 0 to 24.

.. code-block:: shell

   uart:~$ wifi_radio_test bypass_reg_domain 1
   uart:~$ wifi_radio_test tx_power <X>

The TX power cannot be set below :math:`TX_\text{max}` without disabling the regulatory domain control.

.. note::
   For regulatory certification type testing, both regulatory domain control and TX power should use default settings, meaning that the commands described above should not be issued.

TX inter-packet gap
*******************

> The ``wifi_radio_test init`` commands resets the inter-packet gap to ``0``.

Reviewer: ```suggestion
The ``wifi_radio_test init`` commands reset the inter-packet gap to ``0``.
```

   uart:~$ wifi_radio_test he_ltf 2
   uart:~$ wifi_radio_test he_gi 2
   uart:~$ wifi_radio_test tx_pkt_gap 200
   uart:~$ wifi_radio_test tx_pkt_num -1
   uart:~$ wifi_radio_test tx 1

Running a continuous OFDM TX traffic sequence in HE-TB-PPDU 11ax mode
=====================================================================

Set the parameters to run a continuous OFDM TX traffic sequence in HE-TB-PPDU 11ax mode.

* Channel: 100
* Frame format: HE-TB (11ax)
* Payload length: 1024 bytes
* Inter-frame gap: 200 µs
* Data rate: MCS7
* 3.2 µs GI
* 106 Tone
* 4xHELTF
* RU Index 2

This configuration gives a frame duration of 332 µs and an achieved duty-cycle of 62.4%.

Execute the following sequence of commands:

.. code-block:: shell

   uart:~$ wifi_radio_test init 100
   uart:~$ wifi_radio_test tx_pkt_tput_mode 5
   uart:~$ wifi_radio_test tx_pkt_mcs 7
   uart:~$ wifi_radio_test ru_tone 106
   uart:~$ wifi_radio_test ru_index 2
   uart:~$ wifi_radio_test tx_pkt_len 1024
   uart:~$ wifi_radio_test he_ltf 2
   uart:~$ wifi_radio_test he_gi 2
   uart:~$ wifi_radio_test tx_pkt_gap 200
   uart:~$ wifi_radio_test tx_pkt_num -1
   uart:~$ wifi_radio_test tx 1

> At any point in time, the following command can be used to verify the configurations set.

Reviewer: ```suggestion
At any point in time, you can use the following command to verify the configurations set.
```
